### ISSUE TITLE
Reverting to pre XDG-path or Finding config.yaml

### ISSUE THREAD

**Issue by nanooq** [sup-heliotrope/sup#467]
Hello all, 

for testing, I would like to revert back from XDG-paths and put together my $HOME/.sup - folder, which I haven't backuped. I cannot find my old config.yaml in any XDG-path anymore. Where can I find it?

**Comment by gauteh** [comment]
Nanooq writes on August 25, 2015 12:48:
> Hello all, 
> 
> for testing, I would like to revert back from XDG-paths and put together my $HOME/.sup - folder, which I haven't backuped. I cannot find my old config.yaml in any XDG-path anymore. Where can I find it? 
> 

It should be in ~/.config/sup/

**Comment by nanooq** [comment]
Ach, it must be terrible to have a user such as me. 

Anyways, thanks to @gauteh I was able to reconstruct my old .sup-folder. The files needed where found in ~/.config/sup/ and ~/.local/share/sup - but finally I had to fall back on my backup:

Before that mishap, I pulled the new vgit version and started sup I was asked something about moving. I answered yes and was returned to bash. Starting sup again, I discovered, that my ~/.config/sup/config.yaml was reset to default. I am reluctant to open another issue about this behavior: I think the misunderstanding comes from sup asking me doing something manual and me not having a clue what that means.

So, @gauteh, if you agree, I close this issue as my issue as explained above has been fixed (and led me to further insights in sup, which is good)

**Comment by gauteh** [comment]
Hi, let us know if it works. config.yaml should not be reset anyway! 


@ZJvandeWeg: #401 ping!

Nanooq writes on August 25, 2015 18:40:

**Comment by ZJvandeWeg** [comment]
On my radar. Will look into this today.

**Comment by gauteh** [comment]
@nanooq: by any chance, did the correct config.yaml end up in .local/share/sup ?

**Comment by gauteh** [comment]
@ZJvandeWeg: seems my wording on the message could be improved.. suggestions appreciated ;)

**Comment by ZJvandeWeg** [comment]
I just figured out that for example the `signature` file is also moved though this is not reflected in the config.
Thinking about a sentence now

**Comment by gauteh** [comment]
@ZJvandeWeg: what happens if the files already exist at the destination? In case a user reverts to pre-xdg, a new index is created, then moves to post-xdg and the index is moved again?

**Comment by gauteh** [comment]
possibly related #477.

**Comment by schoeke** [comment]
I ran into the same problem.

I would suggest making this kind of warning way wordier! Tell me, what is going to happen if I press "y" and what is going to happen if I press "n". 

"In order to <why are you doing this>, we are about to move your configuration from <source> to <destination>. We want to make sure, that nothing is lost, but we can't make backups for you. If you would like to stop here, take a break and take care of that (e.g. by running  "cp <source> <source.bu> yourself), please press <key> now. If you are fine with moving the files now, press <key>."

Just a quick thought on my side.

**Comment by 97-109-107** [comment]
Since the migration, [the fix (#477) Highline maintainers](https://github.com/sup-heliotrope/sup/issues/477)  gave resolved my issue I reported there.

However, while running 7989b5b with `ruby 2.2.3p173` sup hangs, but not crashes, when I trigger the search mode.

**Comment by gauteh** [comment]
This and #467 is critical. I have reverted the xdg patches and put them in the xdg branch, for git testers: please move all the content of .config/sup/* and .local/share/sup/* back to ~/.sup.

@ZJvandeWeg: We need more testing on this before we can keep it in develop, we can't risk more data loss.
